registry/handlers/hooks.go

```diff
@@ -0,0 +1,26 @@
+package handlers
+
+import (
+	"github.com/Sirupsen/logrus"
+)
+
+// logHook is for hooking Panic in web application
+type logHook struct {
+	LevelsParam []string
+	Mail        *mailer
+}
+
+// Fire forwards an error to LogHook
+func (hook *logHook) Fire(entry *logrus.Entry) error {
+	return hook.Mail.sendMail("Email From Docker Registry", entry.Message)
```

Comment:
I know we are close but we need to make this mail format more useful. Let's add some context to this subject line, such as the level, hostname and the truncated message:

```
[panic] registry0.example.com: <msg>
```

We also want to render the fields (entry.Fields) as part of the email body. The `text/template` package template can be used to do this:

```
{{.Message}}

{{range $key, $value := .Fields}}
{{$key}}: {{$value}}
{{end}}
```